config/routes.rb

```diff
@@ -29,6 +29,9 @@
   resources :inquiries,       only: [:create]
 
   resources :challenges, only: [:show, :index] do
+    member do
+      get 'drawing'
```

Comment:
Just FYI – we can also write this as `:drawing` which I'm a bit partial to just to match all the rest of the routing, but not a huge deal / functionality diff at all.